echo-telemetry/src/main/java/com/netflix/spinnaker/echo/config/TelemetryConfig.java

```diff
@@ -64,6 +68,22 @@ private OkClient telemetryOkClient(TelemetryConfigProps configProps) {
     return new OkClient(httpClient);
   }
 
+  @Bean
+  public CircuitBreaker statsCircuitBreaker(CircuitBreakerRegistry circuitBreakerRegistry) {
+    CircuitBreakerConfig config =
+        CircuitBreakerConfig.custom()
+            .waitDurationInOpenState(Duration.ofHours(12)) // Arbitrary.
+            .ringBufferSizeInClosedState(10)
+            .build();
+
+    return circuitBreakerRegistry.circuitBreaker("stats", config);
+  }
+
+  @Bean
+  public CircuitBreakerRegistry circuitBreakerRegistry() {
```

Comment:
would it be worth adding the resilience4j spring boot starter instead: https://resilience4j.readme.io/docs/getting-started-3

would allow the config options to get pulled from yaml

Reply:
Oy - gotta upgrade to 1.0.0 to fix an issue with pulling in the whole spring boot starter: https://github.com/spinnaker/kork/pull/384

Reply:
Done.